Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1508,7 +1508,7 @@ NOTE: Restart is required for the setting to take effect.</value>
     <comment>{0}: Variable Name; {1}: name of conflicting declaration</comment>
   </data>
   <data name="GroupingGrid_Filter" xml:space="preserve">
-    <value>filter by description</value>
+    <value>Filter</value>
```

Comment:
Could you please delete the values for this resource key from the translated resources? Then we do not forget to translate the new version.